qds_sdk/commands.py

```diff
@@ -87,10 +87,26 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
```

Comment:
In UI we just delete the whole dom and re-populate it with new content. In cli I don't think its possible.